Assets/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "xyz.0xsequence.waas-unity",
-  "version": "2.0.0",
+  "version": "2.1.0",
```

Comment:
Probably 2.0.1 is better until we add a new feature (i.e. Apple Sign-in is ready)

Reply:
I've added a sample, end sessions automatically OnApplicationQuit, and updated supported networks. Shouldn't that be 2.1.0?

Reply:
Nope, assuming we are following semver, unless we add net new features should be smallest number increase.

Reply:
Ok my mistake